pull_request_template.md

```diff
@@ -33,25 +34,11 @@
 
 <!-- Describe the steps to test the changes that this PR introduces; or explain the reason why it is not possible to test the changes -->
 
-## Assignee
-
-<!-- Add mentions to the PR author and also to reviewers who tested the changes in UAT -->
-
-## Screenshots
-
-<!-- Add screenshots, link to Loom videos, or link to wireframes/mockups if they are relevant; e.g.: for frontend work -->
-
 ## Checklist
 
 - Manual tests
   - [ ] Changes were tested locally by the author
-  - [ ] Changes were tested on staging by the author
-  - [ ] Changes were tested on staging by reviewers and reviewers mentioned themselves in the "Assignee" section above
   - [ ] Changes were not tested and the reason was explained in the "How to test" section above
-- Automated tests
-  - [ ] Tests were written where appropriate
-- Labels
-  - [ ] `staging` label was added after this PR was released to staging
+
 - Others
-  - [ ] There are no typos, spelling, or grammatical errors in UI/UX text
```

Comment:
Hi @CaptainAwesomeDi the grammatical error and automated test checkboxes as not used a lot but I believe they serve as a reminder

Reply:
I believe typos can be avoided by using spellcheck plugin, automate tests shouldn't be a reminder but a habit